packages/jest-cli/src/reporters/__tests__/coverage_reporter.test.js

```diff
@@ -97,6 +104,28 @@ describe('onRunComplete', () => {
       });
   });
 
+  it('getLastError() returns an error when threshold is not met for file', () => {
+    const testReporter = new CoverageReporter(
+      {
+        collectCoverage: true,
+        coverageThreshold: {
+          '/full/file/path.js': {
+            statements: 100,
+          },
+        },
+      },
+      {
+        maxWorkers: 2,
+      },
+    );
+    testReporter.log = jest.fn();
+    return testReporter
+      .onRunComplete(new Set(), {}, mockAggResults)
+      .then(() => {
+        expect(testReporter.getLastError()).toBeTruthy();
```

Comment:
It would be nice to at least once assert on what's the content of the error, what do you think?